gateway-protocol-impl/pom.xml

```diff
@@ -22,7 +22,7 @@
   <properties>
     <version.java>8</version.java>
     <license.header>com/mycila/maven/plugin/license/templates/APACHE-2.txt</license.header>
-    <proto.dir>${project.build.directory}/proto</proto.dir>
+    <proto.dir>${maven.multiModuleProjectDirectory}/gateway-protocol/src/main/proto</proto.dir>
```

Comment:
❓ Does it maintain module build order? Or more importantly: Does the gateway-protocol even need to be build before the gateway-protocol-impl module for this?

Reply:
The gateway protocol module doesn't build anything other than provide a JAR which contains the resource (should anyone need it). It doesn't even process it, so in our case it doesn't matter. Do we see a future case where we would need the enforce building the gateway-protocol? In that case I would propose using the remote-resource plugin instead of the dependency plugin, as it's more explicit in its goal of sharing resources via JARs.